docs/machine-learning/tutorials/health-violation-classification-model-builder.md

```diff
@@ -59,86 +60,118 @@ Each row in the dataset contains information regarding violations observed durin
+## Add the code to make predictions
 
-    [!code-csharp [ProgramUsings](~/machinelearning-samples/samples/modelbuilder/MulticlassClassification_RestaurantViolations/RestaurantViolations/Program.cs#L2)]
+## (Optional) Consume the model
 
-1. To make a prediction on new data using the model, create a new instance of the `ModelInput` class inside the `Main` method of your application. Notice that the risk category is not part of the input. This is because the model generates the prediction for it.
+This step will have project templates that you can use to consume the model. This step is optional and you can choose the method that best suits your needs on how to serve the model.
 
-    [!code-csharp [TestData](~/machinelearning-samples/samples/modelbuilder/MulticlassClassification_RestaurantViolations/RestaurantViolations/Program.cs#L11-L15)]
+- Console App
+- Web API
 
-1. Use the `Predict` method from the `ConsumeModel` class. The `Predict` method loads the trained model, creates a [`PredictionEngine`](xref:Microsoft.ML.PredictionEngine%602) for the model, and uses it to make predictions on new data.
+### Console App
 
-    [!code-csharp [Prediction](~/machinelearning-samples/samples/modelbuilder/MulticlassClassification_RestaurantViolations/RestaurantViolations/Program.cs#L17-L24)]
+When adding a console app to your solution, you will be prompted to name the project.
 
+1. Name the console project **RestaurantViolationsPrediction_Console**.
+1. Click **Add to solution** to add the project to your current solution.
 1. Run the application.
 
     The output generated by the program should look similar to the snippet below:
 
     ```bash
-    Inspection Type: Complaint
-    Violation Description: Inadequate sewage or wastewater disposal
-    Risk Category: Moderate Risk
+    InspectionType: Routine - Unscheduled
+    ViolationDescription: Moderate risk food holding temperature
+
+    Predicted RiskCategory: Moderate Risk
     ```
 
-If you need to reference the generated projects at a later time inside of another solution, you can find them inside the `C:\Users\%USERNAME%\AppData\Local\Temp\MLVSTools` directory.
+### Web API
+
+When adding a web API to your solution, you will be prompted to name the project.
+
+1. Name the Web API project **RestaurantViolationsPrediction_API**.
+1. Click *Add to solution** to add the project to your current solution.
+1. Run the application.
+1. Open PowerShell and enter the following code where PORT is the port your application is listening on.
+
+    ```powershell
+    $body = @{
+        InspectionType="Reinspection/Followup"
+        ViolationDescription="Inadequately cleaned or sanitized food contact surfaces"
+    }
+
+    Invoke-RestMethod "https://localhost:<PORT>/predict" -Method Post -Body ($body | ConvertTo-Json) -ContentType "application/json"
+    ```
+
+1. If successful, the output should look similar to the text below:
```

Comment:
I'd add some information here to explain what the output means. Especially the score.

Reply:
Added some info here, but I'm not real sure if it's enough.